docs/en/Overview.md

```diff
@@ -11,8 +11,8 @@ priority: 0
 ## What is Alluxio
 
 Alluxio is the world’s first memory-speed virtual distributed storage system.
-It unifies data access and bridges computation frameworks and their underlying storage systems.
-Applications can access data stored in numerous storage system through a single unified source.
+It bridges the gap between computation frameworks and storage systems, enabling applications
```

Comment:
removed "unifies data access" here? but it sounds so important :O @aaudiber

Reply:
ugh

Reply:
I think it's unclear a less clear way of saying "enabling applications to connect to numerous storage systems through a common interface"